docker/zenml-dev.Dockerfile

```diff
@@ -10,13 +10,10 @@ ENV PYTHONFAULTHANDLER=1 \
     ZENML_LOGGING_VERBOSITY=INFO \
     ZENML_CONTAINER=1
 
-
-WORKDIR /zenml
```

Comment:
Why is copying to `/` preferable to having a separate directory where the ZenML files are located?

Reply:
It's not, you're right. I went too far trying to make the dev image resemble the official base image, but the official base image doesn't copy any files, which is why it doesn't need a workdir.

Reply:
Done